lib/pure/strformat.nim

```diff
@@ -316,16 +316,16 @@ single letter DSLs.
 import macros, parseutils, unicode
 import strutils except format
 
-proc mkDigit(v: int, typ: char): string {.inline.} =
+func mkDigit(v: int, typ: char): string {.inline.} =
   assert(v < 26)
   if v < 10:
     result = $chr(ord('0') + v)
   else:
     result = $chr(ord(if typ == 'x': 'a' else: 'A') + v - 10)
 
-proc alignString*(s: string, minimumWidth: int; align = '\0'; fill = ' '): string =
+func alignString*(s: string, minimumWidth: int; align = '\0'; fill = ' '): string =
   ## Aligns `s` using the `fill` char.
-  ## This is only of interest if you want to write a custom `format` proc that
+  ## This is only of interest if you want to write a custom `format` func that
```

Comment:
```suggestion
  ## This is only of interest if you want to write a custom `format` proc that
```
Here it makes sense to say proc, this is not a definition, ditto for the other cases in doc comments (except for links).

Reply:
Done